--- libopensync-plugin-synce-rra-0.22.2/src/synce_plugin.c.orig	2009-04-27 19:47:45.000000000 +0200
+++ libopensync-plugin-synce-rra-0.22.2/src/synce_plugin.c	2015-02-01 21:20:22.695791287 +0100
@@ -160,7 +160,7 @@
         env->codepage = iconv_codepage;
 
 	//1 - creating matchmaker
-	matchmaker = rra_matchmaker_new();
+	matchmaker = rra_matchmaker_new(NULL);
 	if (!matchmaker){
 		osync_context_report_error(ctx, OSYNC_ERROR_GENERIC, "building matchmaker");
 		return;
@@ -177,7 +177,7 @@
 	osync_debug("SynCE-SYNC", 4, "partner set");
 
         //3 -setting timezone
-        if (!rra_timezone_get(&(env->timezone))){
+        if (!rra_timezone_get(&(env->timezone), NULL)){
 		osync_context_report_error(ctx, OSYNC_ERROR_IO_ERROR, "getting timezone");
 		goto exit;
     	}
